app/cdap/components/AbstractWidget/FormInputs/MultiSelect/index.tsx

```diff
@@ -42,6 +38,20 @@ const styles = (theme) => {
     root: {
       margin: theme.Spacing(2),
     },
+    chip: {
```

Comment:
Okay sorry if I wasn't clear, none of these things should be done this way. We decided _new_ stuff uses styled components and all these are new. If you can't get something to work, like the tooltip, its fine to do that the other way but you should add a comment mentioning that. Chip and hyperlink should be done using styled components

Reply:
made the changes